### DISCUSSION TITLE
Reference documentation

### DISCUSSION THREAD

**Discussion by dafvid** [nuejs/nue#37] [Q&A]
Code examples are nice and all that but are there any reference documentation anywhere? Arguments to functions etc explaning all the bits and pieces. I tend to switch between them both when learning something new.

**Comment by tipiirai** [comment]
True. The API methods are currently mixed inside the docs. Should be separated.

**Reply by dafvid** [reply]
Yea I kinda figured the bold createApp was a link to the reference docs =) It's really handy when it is.
